### DISCUSSION TITLE
Clearing a cookie when logout() on subdomain

### DISCUSSION THREAD

**Discussion by pashinin** [maxcountryman/axum-login#178] [Q&A]
There is a check in integration tests that a cookie is destroyed on logout. And it is working until you use a subdomain.

For example:

```
auth.example.localhost - sets a cookie "id" for a domain ".example.localhost"
```

Then on logout this cookie is not cleared because there is no "Domain" param in a response. 

```
Set-Cookie: id=; Max-Age=0; Expires=Mon, 06 Feb 2023 09:20:24 GMT
```

So a cookie is not removed from browser causing warnings:

```
tower_sessions_core::session: possibly suspicious activity: record not found in store
```

**Comment by maxcountryman** [comment]
Please see [this tower-sessions discussion thread](https://github.com/maxcountryman/tower-sessions/discussions/153).
